src/components/Book.jsx

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable linebreak-style */
+// eslint-disable-next-line import/no-extraneous-dependencies
+import PropTypes from 'prop-types';
+
```

Comment:
I am hightlighting the `import/no-extraneous-dependencies` linter disabling here
- [ ] Linters help us write clean code. Let's maintain that. It is not a good practice to use packages without adding them. The `prop-types` dependency does not exist in your package file. Since you are using it then you should add it - as suggested in the linter error message when linter is enabled. Kindly run the **`npm i -S prop-types`** as described in the message to do that.

Reply:
Ok, bro. Thanks for your help bro.
✔️ prop-types package installed and configured